### PR TITLE
Removed token, adding user 

### PR DESCRIPTION
**Pull request by darkyldull**
Added user DM, to spam inboxes

Just for practicing, forgot git commands and experimenting with classes

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 from discord.ext import tasks, commands
 
 token = ""
+emoji = '💦'
+
```
Comment (aksdad) [inline bot.py:6]: nit: extra line

File: bot.py [hunk 2]
```diff
@@ -10,13 +12,18 @@ async def on_ready(self):
...
         self.water_printer.start()
-    
+
     @tasks.loop(seconds=10.0)
     async def water_printer(self):
         message_channel = self.get_channel(729573884722151486)
-        print('sending message to channel ' + '729573884722151486: ' + message_channel.name + '\n')
-        await message_channel.send('drink water pls domo arigatou - Sahil \'Darky\' Guleria')
-
+        user = self.get_user(219903949145243648)
```
Comment (aksdad) [inline bot.py]: might wanna remove your user id

**Review by aksdad — CHANGES_REQUESTED** [review]

File: bot.py (hunk 1 above)
Comment (darkyldull) [reply bot.py:6]: removed

File: bot.py (hunk 2 above)
Comment (darkyldull) [reply bot.py]: removed

**Review by aksdad — APPROVED** [review]

**Pull request merged**